examples/apps/weather/src/Error.flix

```diff
@@ -0,0 +1,24 @@
+mod Error{
```

Comment:
I think We don't need this namespace because all errors are WeatherError right? So we just need the WeatherError namespace.